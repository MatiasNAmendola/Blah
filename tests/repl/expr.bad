~
*5
3 +--5
51 * True
51 * (True)
-False
-(False)
-
3
5)
7/0
a:
12
b
:3
ad * 1
a: b: e
a + b: 7
'stuff' & and True
[,]
['hello','I','am',]
'unended string
