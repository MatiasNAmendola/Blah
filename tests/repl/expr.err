^<tokenize>
^<parse>
^<parse>
^<eval>
^<eval>
^<parse>
^<eval>
^<parse>
^3$
^<parse>
^<eval>
^<parse>
^12$
^<eval>
^<parse>
^<eval>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<tokenize>
