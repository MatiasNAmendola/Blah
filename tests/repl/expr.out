<tokenize> umatched token '~'
<parse> [Dk MULT] [INT 5]
<parse> [Dl (Ls (Se (Oa (Ac (Cj (Ja (At (Tf (Fn (Ni 5)))))))))),Dk MINUS,Dk PLUS,Da (At (Tf (Fp (Ni 3))))] []
<eval> multiplication of "51" and "True" is not supported
<eval> multiplication of "51" and "True" is not supported
<parse> [Dl (Ls (Se (Oa (Ac (Cj (Ja (At (Tf (Fb False))))))))),Dk MINUS] []
<eval> negation of "False" is not supported
<parse> [Dk MINUS] []
3
<parse> [Dl (Ls (Se (Oa (Ac (Cj (Ja (At (Tf (Fp (Ni 5))))))))))] [RPAREN]
<eval> divide by zero
<parse> [Dk COLON,Dn (Nd "a")] []
12
<eval> variable "b" does not exist
<parse> [Dk COLON] [INT 3]
<eval> variable "ad" does not exist
<parse> [Dl (Ls (Sa (AssnId "b" (Oa (Ac (Cj (Ja (At (Tf (Fp (Nd "e"))))))))))),Dk COLON,Dn (Nd "a")] []
<parse> [Dl (Ls (Sa (AssnId "b" (Oa (Ac (Cj (Ja (At (Tf (Fp (Ni 7))))))))))),Dk PLUS,Da (At (Tf (Fp (Nd "a"))))] []
<parse> [Dk AND,Dk CONCAT,Dj (Ja (At (Tf (Fs "stuff"))))] [TRUE]
<parse> [Dk COMMA,Dk LBRA] [RBRA]
<parse> [Dk RBRA,Dk COMMA,Di (Lcons (Lcons (Lone (Oa (Ac (Cj (Ja (At (Tf (Fs "hello")))))))) (Oa (Ac (Cj (Ja (At (Tf (Fs "I")))))))) (Oa (Ac (Cj (Ja (At (Tf (Fs "am")))))))),Dk LBRA] []
<tokenize> unterminated string
